.github/workflows/security-audit.yml

```diff
@@ -13,13 +13,21 @@ jobs:
         branch: ["v2"] # List of branches to run the security audit uppon
 
     steps:
+      - name: Checkout the default branch (security configuration)
+        uses: actions/checkout@v3
+
       - name: Checkout '${{ matrix.branch }}' branch
         uses: actions/checkout@v3
         with:
           ref: ${{ matrix.branch }}
+          path: ${{ matrix.branch }}
```

Comment:
Will this path be located inside the checked-out default branch? In that case, I think this could lead to issues when the branch name conflicts with an existing directory in the repository.

Maybe we could configure a sub-directory for the initial checkout too? Something like "default"?

Reply:
Thanks!